package.json

```diff
@@ -1,65 +1,77 @@
 {
   "name": "quilt",
   "version": "1.0.0",
+  "private": true,
   "description": "A loosely related set of packages for JavaScript/TypeScript projects at Shopify",
-  "scripts": {
-    "publish": "lerna publish",
-    "build": "yarn tsc --build packages",
-    "lint": "eslint '**/*.{ts,tsx}'",
-    "lint:changelogs": "node scripts/lint-changelogs.js",
-    "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
-    "_test": "NODE_ICU_DATA=node_modules/full-icu jest",
-    "pretest": "yarn build",
-    "test": "yarn _test --no-coverage",
-    "test:ci": "yarn _test --maxWorkers=3 --coverage",
-    "check": "lerna run check",
-    "release": "yarn lint:changelogs && lerna publish",
-    "release-beta": "lerna publish --dist-tag beta",
-    "clean": "rimraf './packages/*/dist'",
-    "generate": "plop",
-    "generate:package": "plop package && yarn plop docs && prettier --write \"./packages/tsconfig.json\"",
-    "tophat": "tophat"
+  "homepage": "https://github.com/shopify/quilt#readme",
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Shopify/quilt.git"
   },
-  "author": "Shopify Inc.",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/shopify/quilt/issues"
-  },
-  "homepage": "https://github.com/shopify/quilt#readme",
-  "private": true,
+  "author": "Shopify Inc.",
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "_test": "NODE_ICU_DATA=node_modules/full-icu yarn sk test",
+    "build": "sk build",
+    "check": "lerna run check",
+    "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
+    "clean": "rimraf './packages/*/build' './packages/*/*.d.ts' './packages/*/*.js' './packages/*/*.esnext' './packages/*/*.mjs' '.sewing-kit'",
```

Comment:
Clearing all these files manually feels like a guessing game to me. We should rely on sewing-kit to handle this. 

There are a number of files that match the following  `'./packages/*/*.d.ts' './packages/*/*.js'` that are removed in this PR. Did any of these get accidentally picked up by a `yarn run clean` or are all of them covered by sewing-kit's `pkg.entry` automation?

I don't expect an answer for this 😅 Just more things to consider during our validation of sewing-kit-next.

Reply:
I think I deliberately got rid of some `packages/*/*.d.ts` and `packages/*/*.js` files (also added them to `.gitignore`) since we didn't need them anymore with our new generated entrypoints, but it's probably worth it for me to go back and double-check that they were all actually just entrypoints 👍